seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-1/src/test/resources/jdbc_mysql_source_and_sink.conf

```diff
@@ -22,13 +22,17 @@ env {
 
 source {
   jdbc {
-    url = "jdbc:mysql://mysql-e2e:3306/seatunnel?useSSL=false"
+    url = "jdbc:mysql://mysql-e2e:3306/seatunnel"
     driver = "com.mysql.cj.jdbc.Driver"
     connection_check_timeout_sec = 100
     user = "root"
     password = "Abc!@#135_seatunnel"
 
     query = "select * from source;"
+     properties {
+       useSSL=false
+       rewriteBatchedStatements=true
```

Comment:
` rewriteBatchedStatements=true` Can you add this configuration to option by default? If the user does not set it manually, we can also enable this optimization in the demo

Reply:
I have already added it. Please take a look and let me know if it meets the requirements.